remotespark/kernels/kernelmagics.py

```diff
@@ -174,11 +175,12 @@ def sql(self, line, cell="", local_ns=None):
     @needs_local_scope
     @argument("-o", "--output", type=str, default=None, help="If present, query will be stored in variable of this "
                                                              "name.")
+    @argument("-q", "--quiet", type=bool, default=False, const=True, nargs="?", help="Do not display visualization")
```

Comment:
Good point. Should I edit the "help" to be something like "return None instead of a pandas df"?